custom_components/tahoma/const.py

```diff
@@ -37,34 +37,6 @@
     WATER_HEATER,
 ]
 
-IGNORED_OVERKIZ_DEVICES = [
-    "ProtocolGateway",
```

Comment:
We should keep this, however we need to use this in a different way. In another PR I face issues where we create a device entry for `Pod` and `ProtocolGateway`, which we want to ignore.

However, the other entries could be removed indeed! :)